tests/kernel/fpu_sharing/generic/src/float_regs_arm_gcc.h

```diff
@@ -1,6 +1,6 @@
 /**
  * @file
- * @brief ARM Cortex-M4 GCC specific floating point register macros
+ * @brief ARM Cortex-M4/R4 GCC specific floating point register macros
```

Comment:
This is no longer correct.

Let's just make it:
```
ARM GCC specific floating point register macros
```

Reply:
done